doc/source/examples/analysis/custom_parallel_analysis.ipynb

```diff
@@ -71,12 +71,14 @@
   {
```

Comment:
<p>This seems to be a vastly different image, do we know why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>Pinging this again, the radius of gyration output here doesn't seem right, these shouldn't be straight lines?</p>